git-hg/setup10.sh

```diff
@@ -21,4 +21,4 @@ git init
 git checkout -b root-commit || exit 1
 git remote add github git@github.com:AdoptOpenJDK/openjdk-jdk10.git
 cd - || exit 1
-bash add-branch.sh jdk10/jdk10
+bash add-branch.sh jdk/jdk
```

Comment:
Why has this changed to `jdk` instead of `jdk10`?

Reply:
@Shura has shared this with us:
> As per @shura, the new announcement points to this change - http://mail.openjdk.java.net/pipermail/jdk-dev/2017-December/000388.html

Reply:
right okay